README.md

```diff
@@ -73,6 +75,13 @@ pod 'LayoutKit'
 ```
 
 Then run `pod install`.
+### Carthage
+To install with Carthage, follow the instruction on Carthage
```

Comment:
Can you add a link to Carthage (like I did for Cocoapods), and end the sentence with a period.